### PR TITLE
OSM-Seed Tiler server

### PR DESCRIPTION
**Pull request by Rub21**
Those are the changes to create  the tiler server for osm-seed

**There are 3 containers in order to run it the tiles server:**
- Tiler-db: contains the PostGIS database 
- Tiler-imposm: This a container ins charge to import and update the data into the PostGIS DB.
- Tiler-server: Tegola server for tiling. 

cc. @batpad @kamicut @geohacker

### CODE REVIEW

**Comment by batpad** [comment]
@Rub21 great work :clap: 

 - Can we add all the new images to the CircleCI tests?
 - Should we start talking about / thinking about the Kubernetes / Helm config for this? Am okay to do this as a separate PR.

Would love to chat about this - get some clear directions from you on testing the setup and giving it a shot, and working to merge this as soon as possible. This looks great!

File: tiler-imposm/start.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+mkdir -p /tmp
+
+FILE='http://download.geofabrik.de/south-america/peru-latest.osm.pbf'
```
Comment (geohacker) [inline tiler-imposm/start.sh]: @Rub21 is this importing a pbf file and not from the osm-seed API database? @batpad

Comment (Rub21) [reply tiler-imposm/start.sh]: It is only for test, I will add more code to get the osm-seed replication data and put into the postgis DB.

File: tiler-visor/start.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+sed -i -e 's/demo.tegola.io/'$TILER_SERVER_HOST:$TILER_SERVER_PORT'/g' /usr/share/nginx/html/config.json
+sed -i -e 's/localhost:9090/'$TILER_SERVER_HOST:$TILER_SERVER_PORT'/g' /usr/share/nginx/html/capabilities/osm.json
+sed -i -e 's,https://osm.tegola.io,'${http://$TILER_VISOR_HOST:$TILER_VISOR_PORT}',g' /usr/share/nginx/html/config.json
```
Comment (batpad) [inline tiler-visor/start.sh]: @Rub21 at some point we will want these to be `https` - can we make `$TILER_VISOR_PROTOCOL` another parameter that will define `http` or `https` ?

**Comment by Rub21** [comment]
Closing this PR in favor of : https://github.com/developmentseed/osm-seed/pull/131

**Pull request closed without merging**